examples/shard-worker.config.example

```diff
@@ -116,6 +120,22 @@ redis_shard_backplane_config: {
 
   # The size of the redis connection pool
   jedis_pool_max_total: 4000
+  
+  # the operation queue can be configured to store
+  # operations in specific queues based on platform properties
+  #provisioned_queues: {
+  #  queues: [
+  #    {
+  #      name: "queue1"
+  #      platform: {
+  #        properties: [{name: "key" value: "value"}]
+  #      }
+  #    },
+  #    {
+  #        name: "queue2"
+  #    }
+  #  ]
+  #}
```

Comment:
Place this in line with the example I've provided at the end of the shard-worker with a viable provisioned_queues